src/main/scala/org/broadinstitute/hail/expr/AST.scala

```diff
@@ -1265,6 +1258,10 @@ case class ApplyMethod(posn: Position, lhs: AST, method: String, args: Array[AST
           }
       }
 
+    case (agg: TAggregable, "map", Array(Lambda(_, param, body))) => () => null
+
+    case (agg: TAggregable, "filter", Array(Lambda(_, param, body))) => () => null
```

Comment:
I don't quite get these.  Shouldn't it be an error to evaluate map and filter on aggregables?